src/javascript/app/base/header.js

```diff
@@ -561,12 +618,95 @@ const Header = (() => {
+                        $('.account__switcher-select_currencies').css('display', 'block');
+
+                        const header_deposit = $('.header__deposit');
+                        header_deposit.text('Set currency');
+                        header_deposit.attr('href', Url.urlForDeriv('redirect', `action=add_account&ext_platform_url=${ext_platform_url}`));
+                    }
+
+                    if (is_current && !is_real) {
+                        $(demo_batch1).insertAfter('.header__acc-display');
+                    }
+
+                    // append icons
+                    icon_container.appendChild(account_icon);
+                    // append content
+                    account_content.appendChild(account_text);
+                    account_content.appendChild(account_currency);
+                    account_content.appendChild(account_balance);
+                    // append icons and content
+                    account.appendChild(icon_container);
+                    account.appendChild(account_content);
+                    
+                    if (!is_real) account.appendChild(demo_batch2);
+
+                    account_list.appendChild(account);
+                }
+
+                applyToAllElements('.wallet__switcher-acc', (el) => {
+                    el.removeEventListener('click', loginIDOnClick);
+                    el.addEventListener('click', loginIDOnClick);
+                });
+            });
+        });
+    };
+
     const bindHeaders = () => {
-        const high_risk_accounts_accordion_header = document.getElementById('high_risk_accounts');
-        const low_risk_non_eu_accordion_header = document.getElementById('low_risk_accounts_non_eu');
-        const low_risk_eu_accordion_header = document.getElementById('low_risk_accounts_eu');
-        const low_risk_eu_container = document.getElementById('account__switcher-accordion-eu');
-        if (Client.isHighRisk() || isEuCountry()) {
+        const high_risk_accounts_accordion_header = getElementById('high_risk_accounts');
+        const low_risk_non_eu_accordion_header    = getElementById('low_risk_accounts_non_eu');
+        const low_risk_eu_accordion_header        = getElementById('low_risk_accounts_eu');
+        const low_risk_eu_container               = getElementById('account__switcher-accordion-eu');
+        const is_eu                               = Client.get('loginid').startsWith('MF');
+        if (Client.isHighRisk() && is_eu) {
+            high_risk_accounts_accordion_header.style.display = 'none';
+            low_risk_eu_accordion_header.style.display        = 'flex';
+            low_risk_eu_container.style.display               = 'block';
+            $('<div class="account__switcher-seperator" />').insertBefore('#account__switcher-accordion-eu');
+        } else if (Client.isHighRisk() || isEuCountry()) {
```

Comment:
Okay, may i ask the question what the difference between `is_eu` and `isEuCountry()`?

Reply:
is_eu is accounts that start with 'MF' for example. MF1234 or CR1234.

isEuCountry() is based on residence of the account that comes from account_settings in backend.

MF account can exist in low risk accounts such as south africa or botswana etc.  MF accounts are automatically assigned for accounts with residence in EU countries as well.

Account switcher for low risk countries and EU Countries are completely different.

Reply:
i've change is_eu to is_mf_account to avoid confusion.

Reply:
Okay, got it, thank you for explanation